owl/SOMA-ALL.owl

```diff
@@ -19,4 +19,19 @@ Import(<http://www.ease-crc.org/ont/SOMA-STATE.owl>)
 Import(<http://www.ease-crc.org/ont/SOMA-WF.owl>)
 Import(<http://www.ease-crc.org/ont/SOMA.owl>)
 
+############################
```

Comment:
From the point of view of SOMA in the long term, as opposed to preparing a clone repo as a convenient way to bundle supplemental material for a paper, I would prefer if axioms are stored in the relevant SOMA modules (e.g. ACT for these ones).